R/load_ffverse.R

```diff
@@ -48,9 +48,9 @@ load_ff_rankings <- function(type = c("draft", "week", "all")){
 
   url <- switch(
     type,
-    draft = "https://github.com/dynastyprocess/data/raw/master/files/db_fpecr_latest.rds",
-    week = "https://github.com/dynastyprocess/data/raw/master/files/fp_latest_weekly.rds",
-    all = "https://github.com/dynastyprocess/data/master/files/db_fpecr.rds"
+    draft = "https://github.com/dynastyprocess/data/blob/master/files/db_fpecr_latest.rds?raw=true",
+    week = "https://github.com/dynastyprocess/data/blob/master/files/fp_latest_weekly.rds?raw=true",
+    all = "https://github.com/dynastyprocess/data/blob/master/files/db_fpecr.rds?raw=true"
```

Comment:
I've had problems with the blob + url query format in the past, it looks like the problem is related to the `all` parameter not having the `/raw/` string. Would prefer just adding that rather than switching to blob?raw=true
```suggestion
    draft = "https://github.com/dynastyprocess/data/raw/master/files/db_fpecr_latest.rds",
    week = "https://github.com/dynastyprocess/data/raw/master/files/fp_latest_weekly.rds",
    all = "https://github.com/dynastyprocess/data/raw/master/files/db_fpecr.rds"
```